src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -22,12 +22,8 @@ def multiple_channel_session():
         yield simulated_session
 
 
-def test_self_test():
```

Comment:
Seems like the changes in this file are unrelated to the PR. They should be in a separate PR.

Reply:
See #1197 

This will still show as a difference until that one is merged into master.